common/test/common/NavigationTest.scala

```diff
@@ -34,8 +34,8 @@ class NavigationTest extends FlatSpec with Matchers with OptionValues {
   "US full nav" should "not contain 'membership'" in {
     Us.navigation.filter(_.name.title == "membership").isEmpty shouldEqual true
   }
-  "AU full nav" should "not contain 'membership'" in {
-    Au.navigation.filter(_.name.title == "membership").isEmpty shouldEqual true
```

Comment:
Not sure filtering and checking `isEmpty == true` 💨  is the most obvious way to verify presence. Not your code I know so you can ignore my comment.